applications/app/views/fragments/interactiveBody.scala.html

```diff
@@ -67,19 +66,4 @@
             @fragments.contentFooter(interactive, page.related, isPaidContent = isPaidContent)
         </div>
     }
-
-    @if(!Webpack.isSwitchedOn) {
-        <script>
```

Comment:
To quote Mr @SiAdcock in https://github.com/guardian/frontend/pull/14946

> In addition to being remote, interactive boot scripts are also required dynamically. In order to load these scripts, I have inlined the require calls as non-blocking JavaScript. I have used curl.js to request these scripts. This is already happening in Interactive Articles; I have applied the same logic to News Articles and Liveblog Articles.

So yes, this looks like it should be all good 👌